source/wp-content/themes/wporg-showcase-2022/templates/single.html

```diff
@@ -11,6 +11,13 @@
 <p style="margin-top:var(--wp--preset--spacing--20)"><a href="[domain]">[pretty_domain]</a></p>
 <!-- /wp:paragraph -->
 
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|20","right":"var:preset|spacing|20","bottom":"var:preset|spacing|20","left":"var:preset|spacing|20"},"margin":{"top":"var:preset|spacing|30"}}},"backgroundColor":"blueberry-4"} -->
+<div class="wp-block-group has-blueberry-4-background-color has-background" style="margin-top:var(--wp--preset--spacing--30);padding-top:var(--wp--preset--spacing--20);padding-right:var(--wp--preset--spacing--20);padding-bottom:var(--wp--preset--spacing--20);padding-left:var(--wp--preset--spacing--20)"><!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"600"}},"fontSize":"normal","fontFamily":"inter"} -->
+<h2 class="has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:600">More about this site</h2>
```

Comment:
These will probably need to be moved into a pattern for translating. Will do it in a follow up pr.